src/components/Dropdown/Dropdown.tsx

```diff
@@ -48,13 +49,16 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     className,
     activeClassName,
     parentQuery,
+    alwaysSelectOption = false
```

Comment:
do we need to explicitly set the default as false? I think it would have the same behavior whether `alwaysSelectOption` is false or undefined